spec/src/main/doc/events.asciidoc

```diff
@@ -425,11 +472,36 @@ Any observer method called before completion of a transaction may call +setRollb
 Observer methods may throw exceptions:
 
 * If the observer method is a transactional observer method, any exception is caught and logged by the container.
+* If the observer method is asynchronous, the exception aborts processing of the observer. Exception management during an asynchronous event is defined in <<async_exception>>.
```

Comment:
What about other asynchronous observers in such case? Must, may or mustn't they be aborted also?